joern-cli/frontends/pysrc2cpg/src/main/scala/io/joern/pysrc2cpg/PythonTypeRecovery.scala

```diff
@@ -203,7 +203,7 @@ private class RecoverForPythonFile(cpg: Cpg, cu: File, builder: DiffGraphBuilder
           .foreach { cls =>
             val clsPath = classMethod.typeDecl.fullName.toSet
             symbolTable.put(LocalVar(cls.name), clsPath)
-            if (cls.typeFullName == "ANY")
+            if (cls.typeFullName == Constants.ANY)
```

Comment:
```suggestion
            if (cls.typeFullName == Defines.Any)
```

Reply:
Would it make sense to have `Constants.ANY = Defines.ANY` or maybe remove `Constants.ANY` in favour of `Defines.Any`?